Content/Maps/testmap/setupData.json

```diff
@@ -1,10 +1,10 @@
 {
   "OffsetUnitStartPositionTeamA":[
-  {"unitId":1, "xpos": 2, "ypos": 3},
-  {"unitId":1, "xpos": 3, "ypos": 4}
+  {"unitId":"guardsman", "xpos": 2, "ypos": 3},
```

Comment:
what other metadata can we put in this object? I'm imagining we'll use this same json format for base levels and also save files, so we should plan to be able to store things like movement state (moved/fired or not, etc), current HP, fuel/ammo etc.

Reply:
that's a good point, I didn't think about the idea of damaged units, different states, etc. I will update this when I implement those systems.